app-tasks/.gitignore

```diff
@@ -0,0 +1,16 @@
+# Distribution / packaging
```

Comment:
The `var/` and `lib/` ignore rules here were preventing me from using the directory structure I wanted to use inside of the `statsd/` top level directory. Because they are Python specific, I removed them from the top level `.gitignore` and placed them here.